docs/getting-started-tutorial/4-css-selector-transforms.adoc

```diff
@@ -0,0 +1,171 @@
+:idprefix:
+:idseparator: -
+:toc: right
+:toclevels: 2
+
+# CSS Selector Transforms
+
+Because Lift operates by transforming HTML trees, it needs an easy way to
```

Comment:
You use `it needs` in this sentence, and then talk about things `we'd be doing` in the following sentence. It might be work considering changing this to `we need`, instead.